### PR TITLE
Skyline transfer

### PR DESCRIPTION
**Pull request by rroutsong**
This branch was intended for transfer to skyline hpc, however a few additional updates have been wrapped up in this as well:

- Skyline HPC testing and verification
- FastQC bug fix related to lscratch + filesystem bug
- Bug fixing thanks to @ewalt98 's testing

### CODE REVIEW

**Review by ewalt98 — APPROVED** [review]
I tested the relevant changes, and everything looks good so far.

File: workflow/qc.smk [hunk 1]
```diff
@@ -41,15 +42,22 @@ rule fastqc_trimmed:
...
+        disk_mb = int(500e3) if config['use_scratch'] else 0,
     log: config['out_to'] + "/logs/" + config["project"] + "/fastqc_trimmed/{sids}_R{rnums}.log"
     shell:
         """
+        if [ ! -d "{params.tmp_dir}" ]; then mkdir -p "{params.tmp_dir}"; fi
+        tmp=$(mktemp -d -p "{params.tmp_dir}")
         mkdir -p {params.output_dir}
-        fastqc -o {params.output_dir} -t {threads} {input.in_read}
+        fastqc -o {params.tmp_dir} -t {threads} {input.in_read}
+        find "{params.tmp_dir}" -type f \\( -name '*.html' -o -name '*.zip' \\) \\-exec cp {{}} {params.output_dir} \\;
```
Comment (skchronicles) [inline workflow/qc.smk:59]: Can you update the fastqc and find commands to this?
```
fastqc -o "${{tmp}}" -t {threads} {input.in_read}
find "${{tmp}}" -type f \\( -name '*.html' -o -name '*.zip' \\) -exec cp {{}} {params.output_dir} \\;
```

**Review by skchronicles — CHANGES_REQUESTED** [review]

File: workflow/qc.smk (hunk 1 above)
Comment (rroutsong) [reply workflow/qc.smk:59]: should it be `tmpdir`? https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#standard-resources

**Comment by rroutsong** [comment]
@skchronicles changes sent

**Review by skchronicles — COMMENTED** [review]
Thank you for the updates!

**Review by skchronicles — APPROVED** [review]

**Pull request merged**